src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs

```diff
@@ -191,6 +195,39 @@ impl Sandbox for VirtSandbox {
             .context("create sandbox")?;
 
         inner.state = SandboxState::Running;
+
+        // get and store guest details
+        let guest_details = self
```

Comment:
I wonder if we can move the new contents into a new function, instead of writing them in the `start` function?

Reply:
Thanks, fixed